docs/source/get_involved/code_of_conduct.rst

```diff
@@ -168,15 +168,17 @@ GitHub.
 3.1 Contact points
 ~~~~~~~~~~~~~~~~~~
 
-If you feel able to, please contact Dr Franz Király by email at
-f.kiraly@ucl.ac.uk.
+Please report CoC incidents via the [reporting tool in the issue tracker](https://github.com/alan-turing-institute/sktime/issues/new/choose).
```

Comment:
this is a turing link, and the link is broken. https://github.com/sktime/sktime/issues/3592 Even if fixed, it simply links back to this document, there is no reporting tool I can see